### PR TITLE
S2t1 project classes

### PR DESCRIPTION
**Pull request by nesclark**

### CODE REVIEW

File: src/Entities/ProjectEntity.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace ProjMange\Entities;
+
+use phpDocumentor\Reflection\Types\Boolean;
```
Comment (Steph-ski) [inline src/Entities/ProjectEntity.php]: this can now be deleted as not being used

File: src/Entities/ProjectEntity.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    public function jsonSerialize(): array
+    {
+        return [
+            'id' => $this->id,
+            'name' => $this->name,
+            'client_id' => $this->client_id,
+            'deadline' => $this->deadline,
+            'overdue' => $this->calculateOverdue()
+        ];
+
```
Comment (Steph-ski) [inline src/Entities/ProjectEntity.php:36]: extra space to be removed?

File: src/Entities/ProjectEntity.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
```
Comment (Steph-ski) [inline src/Entities/ProjectEntity.php:38]: extra space to be removed?

**Review by Yas-Shen — COMMENTED** [review]
Looks good here, I agree with Steph otherwise.

File: src/Entities/ProjectEntity.php (hunk 1 above)
Comment (nesclark) [reply src/Entities/ProjectEntity.php]: all done

File: src/Entities/ProjectEntity.php (hunk 2 above)
Comment (nesclark) [reply src/Entities/ProjectEntity.php:36]: all done

File: src/Entities/ProjectEntity.php (hunk 3 above)
Comment (nesclark) [reply src/Entities/ProjectEntity.php:38]: all done

**Comment by randallcoding** [comment]
Agreed with everyone else, docblocks are concise and the code looks great

**Pull request merged**